includes/class-amp-travel-blocks.php

```diff
@@ -0,0 +1,127 @@
+			add_filter( 'the_content', array( $this, 'filter_the_content_amp_atts' ), 10, 1 );
+			add_filter( 'wp_kses_allowed_html', array( $this, 'filter_wp_kses_allowed_html' ), 10, 2 );
+			add_filter( 'pre_get_posts', array( $this, 'filter_search_pre_get_posts' ), 10, 1 );
+		}
+	}
+
+	/**
+	 * Replaces data-amp-bind-* with [*].
+	 * This is a workaround for React considering some AMP attributes (e.g. [src]) invalid.
+	 *
+	 * @param string $content Content.
+	 * @return mixed
+	 */
+	public function filter_the_content_amp_atts( $content ) {
+		return preg_replace( '/\sdata-amp-bind-(.+?)=/', ' [$1]=', $content );
+	}
+
+	/**
+	 * Enqueue editor scripts.
+	 */
+	public function enqueue_editor_scripts() {
+
+		// Enqueue JS bundled file.
+		wp_enqueue_script(
+			'travel-editor-blocks-js',
+			get_template_directory_uri() . '/assets/js/editor-blocks.js',
+			array( 'wp-i18n', 'wp-element', 'wp-blocks', 'wp-components', 'wp-api' )
+		);
+
+		wp_localize_script(
+			'travel-editor-blocks-js',
+			'travelGlobals',
+			array(
+				'themeUrl' => esc_url( get_template_directory_uri() ),
+				'siteUrl'  => site_url(),
+			)
+		);
+
+		// This file's content is directly copied from the Travel theme static template.
+		// @todo Use only style that's actually needed within the editor.
+		wp_enqueue_style(
+			'travel-editor-blocks-css',
+			get_template_directory_uri() . '/assets/css/editor-blocks.css',
+			array( 'wp-blocks' )
+		);
+	}
+
+	/**
+	 * Add the amp-specific html tags required by theme to allowed tags.
+	 *
+	 * @param array  $allowed_tags Allowed tags.
+	 * @param string $context Context.
+	 * @return array Modified tags.
+	 */
+	public function filter_wp_kses_allowed_html( $allowed_tags, $context ) {
+		if ( 'post' === $context ) {
+			$amp_tags = array(
+				'amp-img'  => array_merge( $allowed_tags['img'], array(
+					'attribution' => true,
+					'class'       => true,
+					'fallback'    => true,
+					'heights'     => true,
+					'media'       => true,
+					'noloading'   => true,
+					'on'          => true,
+					'placeholder' => true,
+					'srcset'      => true,
+					'sizes'       => true,
+					'layout'      => true,
+				) ),
+				'amp-list' => array(
+					'class'            => true,
+					'credentials'      => true,
+					'placeholder'      => true,
+					'noloading'        => true,
+					'on'               => true,
+					'items'            => true,
+					'max-items'        => true,
+					'single-item'      => true,
+					'reset-on-refresh' => true,
+					'src'              => true,
+					'[src]'            => true,
+					'width'            => true,
+					'height'           => true,
+					'layout'           => true,
+					'fallback'         => true,
+				),
+			);
+
+			$allowed_tags = array_merge( $allowed_tags, $amp_tags );
+		}
+		return $allowed_tags;
+	}
+
+	/**
+	 * Modify the default search query to include 'adventure' post type.
+	 *
+	 * @param object $query  Original query.
```

Comment:
Could you please remove the extra space between `$query` and `Original query`. It's hard to see, but there are 2 spaces.